decryptor/postgresql/utils.go

```diff
@@ -238,6 +387,9 @@ func NewExecutePacket(data []byte) (*ExecutePacket, error) {
 	return &ExecutePacket{portal, maxRows}, nil
 }
 
+const maxUint16 = 1<<16 - 1
```

Comment:
why not to use `math.MaxUint16`?

Reply:
> why not to use `math.MaxUint16`?

Thanks. I missed these constants. Will change.